packages/react-dom/package.json

```diff
@@ -25,23 +25,28 @@
   "files": [
     "LICENSE",
     "README.md",
-    "index.js",
     "client.js",
+    "client.react-server.js",
+    "index.js",
     "profiling.js",
-    "server.js",
+    "profiling.react-server.js",
+    "react-dom.react-server.js",
+    "server-rendering-stub.js",
     "server.browser.js",
+    "server.bun.js",
     "server.edge.js",
+    "server.js",
     "server.node.js",
-    "server.bun.js",
-    "static.js",
+    "server.react-server.js",
     "static.browser.js",
     "static.edge.js",
+    "static.js",
     "static.node.js",
-    "server-rendering-stub.js",
+    "static.react-server.js",
     "test-utils.js",
     "unstable_testing.js",
+    "unstable_testing.react-server.js",
     "unstable_server-external-runtime.js",
```

Comment:
Updated ordering to match their filesystem layout to make it easier to see if any were missed